docs_src/scheduling.md

```diff
@@ -44,7 +44,7 @@ Clock margin percent
     be specified with **clock period**. This option is equivalent to specifying
     a reduced value for **clock period**.
 
-Clock period relaxation percent:
```

Comment:
curious why this is necessary?

Reply:
I removed the extra ":" and adopted the same format as other options in the doc.